fluentbit/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal AS ubi7
```

Comment:
perconalab/percona-server-mysql-operato:fluentbit-check-old-ubi7                281MB
perconalab/percona-server-mysql-operator:fluentbit-check-ubi8                     319MB

We have additional +38mb please install only needed dependencies

Reply:
It's not that much if we compare base images:
```
registry.access.redhat.com/ubi7/ubi-minimal        81.6MB
redhat/ubi8-minimal                                 102MB
```

Reply:
Ok, with minimal modifications image size is 290MB now:
```
local-fluentbit                               latest                 988c3b187b18   4 seconds ago   290MB
```
I splited microdnf runs to see what is installed for each dependency block and commented out manual install of elfutils-libs and systemd-libs, are they needed?
```diff
@@ -33,15 +33,18 @@ COPY td-agent-bit.repo /etc/yum.repos.d/
 RUN set -ex; \
     microdnf -y update; \
     microdnf -y install \
-        elfutils-libs \
+        #elfutils-libs \
         logrotate \
         procps-ng \
         shadow-utils \
-        systemd-libs \
+        #systemd-libs \
         tar \
-        vim-minimal \
-        percona-xtradb-cluster-client-${PERCONA_VERSION} \
-        postgresql-libs \
+        vim-minimal; \
+    microdnf -y install \
+        postgresql-libs; \
+    microdnf -y install \
+        percona-xtradb-cluster-client-${PERCONA_VERSION}; \
+    microdnf -y install \
         td-agent-bit-${FLUENTBIT_VERSION}; \
     curl -Lf https://github.com/michaloo/go-cron/releases/download/v0.0.2/go-cron.tar.gz -o /tmp/go-cron.tar.gz; \
     tar xvf /tmp/go-cron.tar.gz -C /usr/bin; \
```